core-web/libs/portlets/dot-content-drive/portlet/src/lib/components/dot-content-drive-toolbar/components/dot-content-drive-content-type-field/dot-content-drive-content-type-field.component.ts

```diff
@@ -63,45 +66,67 @@ export class DotContentDriveContentTypeFieldComponent {
         }
     );
 
-    constructor() {
+    ngOnInit() {
         // Set up debounced API request stream with switchMap
         this.#apiRequestSubject
```

Comment:
I suggest moving all this logic + state to a signal store to handle this state consistently.